logging/Differences_between_ODFE_and_OpenSearch.md

```diff
@@ -0,0 +1,97 @@
+# Differences between Open Distro for Elasticsearch and OpenSearch
+
+## Overview
+
+In release 1.2.0 (June 2022), the SAS Viya Monitoring for Kubernetes solution makes the following changes:
+
+* OpenSearch is deployed instead of the Open Distro for Elasticsearch (ODFE) distribution of Elasticsearch.
+* OpenSearch Dashboards is deployed instead of Kibana.
+
+This topic describes the changes between the deployments. It assumes you have an understanding of the deployment process for the current log-monitoring solution including the customization process.
+
+**Note:** Other than the differences described in this topic, you should not see any significant differences when using the log-monitoring solution configured with OpenSearch.
+
+## Important Considerations
+
+When preparing to migrate, be sure to remember the following considerations:
+
+* The migration is nonreversible. You cannot reverse an OpenSearch-based deployment to a deployment that uses Open Distro for Elasticsearch without losing collected log messages.
+* You cannot run an Open Distro for Elasticsearch-based deployment and an OpenSearch-based deployment in the same cluster, even if they are deployed to different Kubernetes namespaces.
+  
+## Differences
+
+### Products
+
+The unstructured data store and search back-end component, formerly Elasticsearch, is now OpenSearch. 
+
+The visualization and reporting application, formerly Kibana, is now OpenSearch Dashboards.
+
+### Helm Charts
+
+Although OpenSearch is virtually identical to Open Distro for Elasticsearch, the Helm chart used to deploy OpenSearch is different. This required the following changes in the customization process: 
+
+* The names of the YAML files used to customize the deployment have changed.
+* The structure of these YAML files has changed. 
+
+See [Compiled Differences](#compiled_dif_table) for more information.
+
+### Deployment Topology
+
+The topology of the search back-end is different. 
+
+* The Open Distro for Elasticsearch search back-end is configured by default to use eight Kubernetes pods composed of two Elasticsearch client nodes, three Elasticsearch master nodes, and three Elasticsearch data nodes.
+* For the OpenSearch search back-end, the default configuration is three Kubernetes pods composed of three OpenSearch "multi-role" nodes.
+* The Java memory settings have increased from 1GB to 4GB for each OpenSearch node.
+
+### NodePort Accessibility
+
+If you had not configured access via Ingress, the Open Distro for Elasticsearch-based deployment script automatically made Kibana accessible via NodePort.
+
+The OpenSearch deployment script does not do this automatically. If you want to make OpenSearch Dashboards accessible via NodePort, you must set the environment variable `KB_KNOWN_NODEPORT_ENABLE` to `True` before running the `logging/bin/deploy_logging_opensearch.sh` script.
```

Comment:
Ugh!  Sorry, I should have caught this earlier: the value of KB_KNOWN_NODEPORT_ENABLE needs to be "true" (all lowercase) rather than "True" )initial capital.
In addition, the script name should be **logging/bin/deploy_logging.sh** rather than **logging/bin/deploy_logging_opensearch.sh**.

Reply:
Changes made. Also, searched the file for any "_opensearch". Now they only occur in the table in the applicable column.